helpers.js

```diff
@@ -16,7 +18,17 @@ function resolveSecurityGroupFunction(ruleType) {
   }
 }
 
+function tryParseJson(v) {
+  if (!_.isString(v)) { return v; }
```

Comment:
If `tryParseJson` doesn't throw an error, we may think that it returned a parsed object, while according to this line it could be a number also.
Please return `v` only if it's an object, otherwise for non-strings throw errors as well.

Unless the filters are meant to accept any value. Are they?